sonar/theme/templates/sonar/footer.html

```diff
@@ -54,8 +54,7 @@
       <div class="col text-center p-0">
         <div class="d-lg-flex flex-row">
           <div class="col-lg text-center text-lg-right">
-            {% set link = 'https://www.rero.ch/de/products/sonar' if current_i18n.locale.language == 'de' else 'https://www.rero.ch/produits/sonar' %}
-            {% trans %}Powered by <a href="{{ link }}" target="_blank" rel="noopener">SONAR</a>{% endtrans %}
+            {% trans %}Powered by <a href="https://github.com/rero/sonar" target="_blank" rel="noopener">SONAR</a>{% endtrans %}
```

Comment:
We need to let the link to [rero.ch](https://www.rero.ch/produits/sonar) since this is the only thing we force to appear on the dedicated portals.